src/components/Header/AccountMenu/AccountMenu.js

```diff
@@ -23,78 +22,83 @@ const formatUsername = (email = '') => {
-                <MenuItem
-                  onClick={logout}
-                  icon={<Icon.SIGN_OUT />}
-                  text={t('header.logout')}
-                />
-              </Menu>
-            </div>
+const AccountMenu = ({
+  t,
+  email,
+  logout,
+  authStatus,
+  togglePrefDialog,
+  toggleExportDialog,
+}) => (
+  <div
+    className={classNames(
+      'account-menu',
+      { 'account-menu--no-email': !authStatus || !email },
+    )}
+  >
+    <Popover
+      minimal
+      position={Position.BOTTOM_LEFT}
+      content={(
+        <div className='account-menu-content'>
+          <Menu>
+            <MenuItem
+              onClick={togglePrefDialog}
+              icon={<Icon.SLIDER_CIRCLE_H />}
+              text={t('header.preferences')}
+            />
+            <MenuItem
+              className='account-menu-export'
+              onClick={toggleExportDialog}
+              icon={<Icon.FILE_EXPORT />}
+              text={t('download.export')}
+            />
+            <MenuItem
+              onClick={openHelp}
+              icon={<Icon.INFO_CIRCLE />}
+              text={t('header.help')}
+            />
+            <MenuItem
+              className='account-menu-sync'
+              shouldDismissPopover={false}
+              text={<SyncMode />}
+            />
+            <MenuItem
+              className='account-menu-query'
+              shouldDismissPopover={false}
+              text={<QueryMode />}
+            />
+            <MenuItem
+              onClick={logout}
+              icon={<Icon.SIGN_OUT />}
+              text={t('header.logout')}
+            />
+          </Menu>
+        </div>
           )}
-          targetTagName='div'
-        >
-          <div className='account-menu-wrapper'>
-            <div className='account-menu-target'>
-              <Icon.USER_CIRCLE />
-              <span className='account-menu-username'>
-                {authStatus ? formatUsername(email) : ''}
-              </span>
-              <Icon.CHEVRON_DOWN />
-              <Icon.CHEVRON_UP />
-            </div>
-          </div>
-        </Popover>
+      targetTagName='div'
+    >
+      <div className='account-menu-wrapper'>
+        <div className='account-menu-target'>
+          <Icon.USER_CIRCLE />
+          <span className='account-menu-username'>
+            {authStatus ? formatUsername(email) : ''}
+          </span>
+          <Icon.CHEVRON_DOWN />
+          <Icon.CHEVRON_UP />
+        </div>
       </div>
-    )
-  }
-}
+    </Popover>
+  </div>
+)
 
-AccountMenu.propTypes = propTypes
-AccountMenu.defaultProps = defaultProps
+AccountMenu.propTypes = {
+  authStatus: PropTypes.bool.isRequired,
+  email: PropTypes.string.isRequired,
+  logout: PropTypes.func.isRequired,
+  t: PropTypes.func.isRequired,
+  toggleExportDialog: PropTypes.func.isRequired,
+  togglePrefDialog: PropTypes.func.isRequired,
+}
 
-export default withTranslation('translations')(AccountMenu)
```

Comment:
by removing the export `withTranslation` you will miss the `t` function from the component props

Reply:
`withTranslation` HOC moved to the related container for better wrappers composition consistency.
So there will be no problem with ` t` providing.

<img width="746" alt="Screenshot 2022-10-19 at 12 33 51" src="https://user-images.githubusercontent.com/41899906/196655195-61fd748e-ca88-47e6-8812-0ec1d4b73c06.png">